exporter/exporterhelper/internal/bounded_memory_queue_test.go

```diff
@@ -23,7 +23,7 @@ import (
 // We want to test the overflow behavior, so we block the consumer
 // by holding a startLock before submitting items to the queue.
 func TestBoundedQueue(t *testing.T) {
-	q := NewBoundedMemoryQueue[string](1)
+	q := NewBoundedMemoryQueue[string](NewNopCapacityLimiter[string](1))
```

Comment:
Probably fake is better than NewNopCapacityLimiter? Of NumRequestsCapacityLimiter which adds 1 per every request.

Reply:
Changed to use `NewRequestsCapacityLimiter`